### ISSUE TITLE
Includes not working with local packages as of 3.3.3/3.4.0-beta1

### ISSUE THREAD

**Issue by ryanfields** [fourseven/meteor-scss#140]
I have a number of local packages in my project under the /packages folder.  Previously, I was just using an index.scss file and including their stylesheets by file path.  Now, I get the same kind of error for all include attempts:

```
While processing files with fourseven:scss (for target web.browser):
   /ondemjobs.scss: Scss compiler error: File to import: {}/packages/views/stylesheets/AppModeChoice.scss not found in file:
   {}/ondemjobs.scss
```

I've also tried ```{views}/stylesheets/AppModeChoice.scss``` without success.

This happens with 3.3.3, and 3.4.0-beta1.

**Comment by sebakerckhof** [comment]
The second syntax `{views}/stylesheets/AppModeChoice.scss` is correct. This is probably due to how you add your scss files in your package.js file.
You need to add them using: api.addFiles([...],'client',{isImport:true})

**Comment by ryanfields** [comment]
In Meteor 1.2 I can't use addFiles for .scss files.  It complains that there's no plugin to handle them, and requires me to use api.addAssets instead.

**Comment by sebakerckhof** [comment]
And you're sure you have fourseven:scss also in your package.js use list for client architecture and you?
Otherwise we'd need a reproduction, because this should work.

**Comment by ryanfields** [comment]
I got this working, but the process was not pleasant.  Doesn't feel like an improvement over the previous version.

Here's how I got it all working:

* Move all of my variables and mixins into a separate package
* api.use that package in all of my local packages that have scss files
* api.use fourseven:scss in all of my local packages with scss to compile the files
* Import the vars/mixins package css into each and every .scss file that uses variables or mixins
* Include each file into the main app scss.

So, where I was previously able to use variables without explicitly including them in my packages, I now have to use explicit imports, at the package level and the scss level.  I'm not sure that's better than before, where I just included the .scss files into a master index file that also imported the vars and mixins.  It was a whole lot less trouble that way.
